tests/YearMonthTest.php

```diff
@@ -386,6 +407,12 @@ public function testPlusYears(int $year, int $month, int $plusYears, int $expect
         $this->assertYearMonthIs($expectedYear, $expectedMonth, $yearMonth->plusYears($plusYears));
     }
 
+    public function testPlusYearsWithParameterIsZero()
```

Comment:
`testPlusZeroYears()`?

Reply:
I agree that and it seems that the test method name can be short.